specs/chains/ethereum/compressed_tx_format.md

```diff
@@ -107,7 +107,7 @@ This document specifies the _compressed_ transaction format, which is posted to
 
 | name   | type                                                                                                                                                                                                                             | description     |
 |--------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|-----------------|
-| `type` | `OutputType`                                                                                                                                                                                                                     | Type of output. |
+| `type` | [`OutputType`](../../protocol/tx_format.md#outputtype)                                                                                                                                                                           | Type of output. |
```

Comment:
[`InputType`](../../protocol/tx_format.md#inputtype) and [`OutputType`](../../protocol/tx_format.md#outputtype) are formatted inconsistently compared with other complex type references e.g.  [OutputCoin](#outputcoin)